rules/macos/persistence_login_logout_hooks_defaults.toml

```diff
@@ -33,8 +33,8 @@ timestamp_override = "event.ingested"
 type = "eql"
 
 query = '''
-process where event.type == "start" and
- process.name == "defaults" and process.args == "write" and process.args in ("LoginHook", "LogoutHook") and
+process where host.os.type == "macos" and event.type == "start" and
+ process.name == "defaults" and process.args == "write" and process.args : ("LoginHook", "LogoutHook") and
```

Comment:
FYSA: I changed the `in` to `:` to circumvent the parsing bug. Should have no negative impact on the logic (and can be changed back once resolved).

REF: #2611

cc: @DefSecSentinel

Reply:
👍